lib/routes/book-router.js

```diff
@@ -2,26 +2,35 @@
 
 const express = require('express');
 const router = express.Router();
+const Users = require('../models/users-model');
+let user = new Users();
+
 
 let books = [
   { title: 'Brave New World', auth: ['admin'] },
   { title: 'Hamlet', auth: ['admin', 'editor'] },
-  { title: 'Alice in Wonderland', auth: ['admin', 'editor', 'user'] }
+  { title: 'Alice in Wonderland', auth: ['admin', 'editor', 'user'] },
 ];
 
 // TODO Swagger Comments
 // TODO Edit code (see lab README)
-router.get('/books', (req, res, next) => {
+router.get('/books', async (req, res) => {
+
   let library = {
     count: books.length,
-    results: books
+    results: books,
   };
-  res.status(200).json(library);
+  let validUser = await user.authenticate(req.body);
+  
+  if(!validUser) {
+    res.status(403).send('This is Restricted Section!!!');
+  } else {res.status(200).json(library);}
+  
 });
 
 // TODO Swagger Comments
 // TODO Edit code (see lab README)
-router.get('/books/:indx', (req, res, next) => {
+router.get('/books/:indx', (req, res) => {
   if (req.params.indx < books.length) {
     let book = books[req.params.indx];
     res.status(200).json(book);
```

Comment:
still need to verify the user is authorized to view certain books and send the 403 error.